storage/ndb/rest-server/rest-api-server/internal/security/apikey/apikey_test.go

```diff
@@ -46,19 +46,12 @@ func TestAPIKey(t *testing.T) {
 	}
 	defer dal.ShutdownConnection()
 
-	databases := []string{testdbs.DB001, testdbs.DB002}
-	err, dropDatabases := testutils.CreateDatabases(true, databases...)
-	if err != nil {
-		t.Fatalf("failed creating databases %v; error: %v ", databases, err)
-	}
-	defer dropDatabases()
```

Comment:
But right now, this test isn't creating any database at all? So if I run this test by itself it will fail?

Reply:
you are right. I forgot to commit a new file. I have pushed "internal/security/apikey/wrapper_test.go"

Reply:
Okay cool. Still though - this means if I want to run this test by itself, it will take 7s longer than without this change. That's okay, but seems a bit unnecessary. I can agree with all other changes, even though I personally prefer running against a Docker cluster, so I don't have the cleanup.

Reply:
if you run any test then there will be a fixed cost at the start for creating all the databases. you have to do this only once until there is a change in the schema.

Reply:
Yeah I get it, it's just 1s (create 3 dbs) vs. 7s (create 20 dbs) in this case